bvm/ballerina-profiler/build.gradle

```diff
@@ -25,7 +25,6 @@ configurations {
 dependencies {
     implementation "org.ow2.asm:asm:${project.ow2AsmVersion}"
     implementation "org.ow2.asm:asm-commons:${project.ow2AsmCommonsVersion}"
-    implementation "com.google.code.gson:gson:${project.gsonVersion}"
```

Comment:
Shall we remove from rt jar as well

Reply:
It is needed for observability right?

Reply:
Resolved through external discussion.

(Fixed in f7f7973d0a9481e6e3070cda49b1baac2efbef5c)